### ISSUE TITLE
use with external drawio installation?

### ISSUE THREAD

**Issue by poVoq** [ONLYOFFICE/plugin-drawio#8]
Is it possible to use an external installation of drawio instead of the build in file?

I guess it would be possible to replace the link in the index.html?

**Comment by ShockwaveNN** [comment]
Hi, is there some advantages in using your own setup of drawio instead of default one?

I don't think we've ever setup our own instance so general speaking - replacing URL in index.html may be a way to do that

**Comment by poVoq** [comment]
Using the latest version and having to maintain only one version. I also use the drawio integration with a selfhosted drawio in Nextcloud directly, so it would make things a bit easier.

I guess I will try and see if replacing it in the index.html works.

What about custom libraries for shapes?

**Comment by ShockwaveNN** [comment]
Sorry, don't understand a question

You mean you've installed custom libraries on your drawio and want them to be available in our plugin so you need a connection to custom server?

**Comment by poVoq** [comment]
Yes, maybe some config option to automatically add custom shape libraries from a remote source?

**Comment by ShockwaveNN** [comment]
I don't think we will dig in such fine details about drawio features

So I advice you to try to replace index.html URL and if this not work - we'll file an issue for our developers

**Comment by ShockwaveNN** [comment]
Oh, we already have an issue 57608 in our private issue tracker to have ability to use local installation of draw.io

But according to devs we're using https://github.com/jgraph/drawio - this plugin and it cannot connect to any other instances

I'll leave this issue opened to gather more user feedback

**Comment by poVoq** [comment]
I am using the same repo. The /SRC/webapp part works as a standalone installation of drawio.

**Comment by ShockwaveNN** [comment]
Ok, thanks. our devs will look at that as soon as they got free time
